backend/globaleaks/utils/users_details_filter.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8
+import re
+
+class UserDetailsFilter:
+    """
+    Filters the text for any user details.
+    Removes things like uuid4, email, pgp formatted strings, session ids.
+    """
+
+    def __init__(self, text):
+        """
+        Filters or scrubs the given text.
+        Arguments:
+            text {string} -- text to be fileted.
+        """
+        self._text = text
+
+    def remove_bgp_formatted_string(self):
```

Comment:
bgp? :)

Reply:
Sorry that was a typo. I've rectified it.